cinema/tests/test_movie_session_api.py

```diff
@@ -67,9 +65,7 @@
         response = self.client.get("/api/cinema/movie_sessions/1/")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.data["movie"]["title"], "Titanic")
-        self.assertEqual(
-            response.data["movie"]["description"], "Titanic description"
-        )
+        self.assertEqual(response.data["movie"]["description"], "Titanic description")
```

Comment:
The test expects the 'genres' and 'actors' fields to be lists of strings. Ensure that the serializer used in the API returns these fields in the expected format. If not, adjust the test or the serializer accordingly.